.github/workflows/documentation.yaml

```diff
@@ -1,37 +0,0 @@
-name: Documentation
-
-on:
-  push:
-    branches:
-      - 'master'
-    paths:
-      - 'lib/**'
-
-jobs:
-  documentation:
-    name: Documentation
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install Rust
-        uses: dtolnay/rust-toolchain@stable
-        with:
-          toolchain: 1.63
-      - name: Install LLVM
-        shell: bash
-        run: |
-          curl --proto '=https' --tlsv1.2 -sSf ${{ env.LLVM_URL }} -L -o llvm.tar.xz
-          mkdir ${{ env.LLVM_DIR }}
-          tar xf llvm.tar.xz --strip-components=1 -C ${{ env.LLVM_DIR }}
-          echo "${{ env.LLVM_DIR }}/bin" >> $GITHUB_PATH
-          echo "LLVM_SYS_120_PREFIX=${{ env.LLVM_DIR }}" >> $GITHUB_ENV
-        env:
-          LLVM_DIR: ${{ github.workspace }}/llvm-13
-          LLVM_URL: 'https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/clang+llvm-13.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz'
-      - name: Build & package documentation
-        run: make package-docs
-      - name: Publish documentation
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./package/docs
```

Comment:
How we are running this now?

Reply:
Note that this is a new workflow because only runs on `master`.

Reply:
@syrusakbary we are running it in a separate file, but we are never shipping the documentation

Reply:
Merged it back into documentation.yaml